### DISCUSSION TITLE
FR: API accessible from windows desktop apps.

### DISCUSSION THREAD

**Discussion by ryantheleach** [File-New-Project/EarTrumpet#631] [Ideas]
EarTrumpet is amazing, I'd be willing to donate to the project, and it's perfect for twitch streamers trying to organize their audio.

However, occasionally we need to automate some actions, including swapping / routing applications to different audio devices.

Exposing an API for EarTrumpet, would make this a ton easier.

**Comment by riverar** [comment]
@ryantheleach Paid for an upgrade? Can you share more information?
